scripts/fork-and-cast-spell.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -e
+
+trap "kill 0" EXIT
+
+spell_path_or_address=$1
+spell_executor=$2
+anvil_fork_url=$3
+anvil_fork_block_number=$4
```

Comment:
last thing: can we make this optional? ie. default value should be latest block

Reply:
done - if not provided we skip --fork-block-number part